### PR TITLE
Make JIRA links really easily

### PR DESCRIPTION
**Pull request by nedbat**
Now you can make JIRA links like this:

```
    :jira:`TNL-4904`
```

### CODE REVIEW

File: shared/conf.py [hunk 1]
```diff
@@ -24,7 +24,16 @@
...
-extensions = ['sphinx.ext.autodoc', 'sphinx.ext.doctest', 'sphinx.ext.intersphinx', 'sphinx.ext.coverage', 'sphinx.ext.pngmath', 'sphinx.ext.mathjax', 'sphinx.ext.ifconfig']
+extensions = [
+    'sphinx.ext.autodoc',
+    'sphinx.ext.doctest',
+    'sphinx.ext.intersphinx',
+    'sphinx.ext.coverage',
+    'sphinx.ext.pngmath',
+    'sphinx.ext.mathjax',
+    'sphinx.ext.ifconfig',
+    'sphinx.ext.extlinks',
```
Comment (nedbat) [inline shared/conf.py:35]: I reformatted the list, and added a new last element.

**Comment by nedbat** [comment]
@pdesjardins @lamagnifica @srpearce @catong A new gizmo for the docs.

**Comment by pdesjardins** [comment]
I like it! 👍

**Comment by lamagnifica** [comment]
nice -- you're right, @nedbat , we'll be able to use this in the release notes! 👍

**Pull request merged**